src/scenes/Authentication/Register/RegisterForm.tsx

```diff
@@ -0,0 +1,128 @@
+} from '../../../components/form';
+import { CordIcon } from '../../../components/Icons';
+import { Link } from '../../../components/Routing';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  header: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  icon: {
+    fontSize: 64,
+    margin: 'auto',
+    marginBottom: spacing(4),
+  },
+  formError: {
+    margin: spacing(2, 0),
+  },
+  submit: {
+    marginTop: spacing(1),
+  },
+  loginLink: {
+    display: 'block',
+    marginTop: spacing(1),
+  },
+  footer: {
+    maxWidth: 500,
+    margin: spacing(3, 0, 2),
+    textAlign: 'center',
+  },
+  footerSpacer: {
+    visibility: 'hidden',
+    '@media (max-height: 700px)': {
+      display: 'none',
+    },
+  },
+  verse: {
+    fontStyle: 'italic',
+  },
+  reference: {
+    marginTop: spacing(1),
+  },
+}));
+
+interface RegisterInput {
+  firstName: string;
+  lastName: string;
+  email: string;
+  password: string;
+}
+
+export type RegisterFormProps = Pick<
+  FormProps<RegisterInput>,
+  'onSubmit' | 'initialValues'
+> & { className?: string };
+
+export const RegisterForm = ({ className, ...props }: RegisterFormProps) => {
+  const classes = useStyles();
+  return (
+    <>
+      <div className={className}>
+        <div className={classes.header}>
+          <CordIcon className={classes.icon} />
+          <Typography variant="h4" gutterBottom={true}>
+            CORD FIELD
+          </Typography>
+          <Typography color="textSecondary">
+            Accelerating Bible Translation
+          </Typography>
+        </div>
+        <Form
+          {...props}
+          decorators={decorators}
+          mutators={{ clearSubmitErrors }}
+        >
+          {({ handleSubmit }) => (
+            <form onSubmit={handleSubmit}>
+              <SubmitError className={classes.formError} />
+              <TextField
+                name="firstName"
+                label="First Name"
+                placeholder="Enter First Name"
+                required
+                autoFocus
+              />
+              <TextField
+                name="lastName"
+                label="Last Name"
+                placeholder="Enter Last Name"
+                required
+              />
+              <EmailField />
+              <PasswordField />
+              <SubmitButton className={classes.submit}>Sign Up</SubmitButton>
+            </form>
+          )}
+        </Form>
+        <Link to="/login" className={classes.loginLink}>
+          Already have an account? Login
+        </Link>
+      </div>
+    </>
+  );
+};
+
+const clearSubmitErrors: Mutator<RegisterInput> = (args, state) => {
+  state.formState = {
+    ...state.formState,
+    submitError: undefined,
+    submitErrors: undefined,
+  };
+};
+
+// decorators get re-created if array identity changes
+// so make constant outside of render function
+const decorators = [blurOnSubmit, focusLastActiveFieldOnSubmitError];
```

Comment:
I think focusFirstFieldWithSubmitError would be better here.
```suggestion
const decorators = [blurOnSubmit, focusFirstFieldWithSubmitError];
```